.env.example

```diff
@@ -50,3 +50,10 @@ DST_REFERENCE_TIMEZONE='US/Eastern'
 PASSPORT_STRATEGY=local
 TEXTER_SIDEBOXES=celebration-gif,default-dynamicassignment,default-releasecontacts,contact-reference,tag-contact,freshworks-widget,default-editinitial,take-conversations,hide-media,texter-feedback
 OWNER_CONFIGURABLE=ALL
+
+
+#PASSPORT_STRATEGY=token
+#TOKEN_AUTH_URL=
+#TOKEN_AUTH_SHARED_SECRET=secret
+#TOKEN_AUTH_ISSUER=issuer
+#TOKEN_AUTH_AUDIENCE=audience
```

Comment:
Can you move this example configuration to the `docs/HOWTO_INTEGRATE_TOKEN_AUTH.md` file?

Reply:
Sure thing!

Reply:
As a matter of fact, it's already there, so I'll just remove from the `.env.example` file